### DISCUSSION TITLE
Video files not loading after reloading website

### DISCUSSION THREAD

**Discussion by esanchezvz** [vercel/next.js#17759] [Help]
# Bug report

## Describe the bug

Using `next@9.5.4` and deployed to vercel. 
I'm trying to display a backgorund video in a landing page for a webapp. This works fine for all browsers except Safari (both mobile and desktop). The background video works fine when you first load the page, but when you refresh or install PWA it no longer loads and only shows the poster. 
This behaviour only happens when the video is hosted on `public/` but this only happens for video files, not any other static file at that directory. If a set a url from a video served from cloudinary (or any other service like AWS S3) background video loads correctly after reloads and after installing pwa. 

At first, I thought this was a problem with caching from `next-pwa` but the issue persists even after disabling pwa functionality.

## To Reproduce

Here is a link to a repository I created to reproduce the issue

[https://github.com/esanchezvz/next-bg-video](https://github.com/esanchezvz/next-bg-video)

and a link to the hosted version on vercel:

[https://next-bg-video.vercel.app/](https://next-bg-video.vercel.app/)

## Expected behavior

Video files being served directly from `public/*`directory should load when refreshing website and installing PWA on every device and browser (it doesn't on Safari)

**Comment by specialfae** [comment]
I believe I'm having the exact same issue with Next hosted on Vercel. 

Video will only play on server side render, any client side renders won't play the video and just show the poster image. 

My quick and dirty solution is to host the video file elsewhere and the problem goes away. 

Very irksome though, I'd much rather have all the site assets bundled in the /public directory. 

Would love a suggestion on how to resolve. 

EDIT: interestingly, my projects with the same issue also have next-pwa installed..

**Reply by specialfae** [reply]
I've moved next-pwa's files from /public to /public/pwa which seems to have helped.
